src/edu/csus/ecs/pc2/core/imports/LoadAccounts.java

```diff
@@ -441,7 +459,7 @@ public Account[] fromTSVFile(String filename, Account[] existingAccounts, Group[
             shortSchoolNameColumn = -1;
             countryCodeColumn = -1;
             teamNameColumn = -1;
-            scoreAdjustmentColumn = -1;
+            institutionCodeColumn = -1;
```

Comment:
This removal does not look right?

Reply:
Yes, it's fine.  It was a duplicate line (see line 454).